_rules/link-in-context-descriptive-5effbb.md

```diff
@@ -35,7 +35,7 @@ This rule applies to any [inheriting semantic][] `link` for which all the follow
 
 ## Expectation
 
-The [accessible name][] of each target element together with its [programmatically determined link context][] describes the purpose of the link.
+The [accessible name][] of each target element together with its [accessible description][] and [programmatically determined link context][] describes the purpose of the link.
```

Comment:
I'm not sure this is obviously correct. The SC talks about link text and programmatically determined context. Its probably worth putting in the assumption that we're using "link text" from WCAG to mean accessible name and description. Links that fail label in name could have visible text that isn't considered "link text" in this rule. I think that should be in the assumption.